docfx.json

```diff
@@ -84,25 +84,25 @@
       },
       {
         "files": [
-          "**/legacy/toc.yml"
+          "**/toc.yml"
```

Comment:
Do we have reference docs on how these fields related to docs.ms page? Or share some information here for my best understanding.
Thanks!